rust/src/collection.rs

```diff
@@ -78,3 +91,76 @@ pub struct Label {
     pub id: i64,
     pub resource_url: String,
 }
+
+fn clean_artist_name(artist: String) -> String {
+    let regex = Regex::new(r"\s*\(\d+\)\s*").unwrap();
```

Comment:
This is just removing whitespace, right? https://doc.rust-lang.org/std/primitive.str.html#method.trim you won't need regex for that.

Reply:
that is not the case. some artists share names or have multiple artist entries. for instance "Electric Wizard (2)". This removed the " (2)" in such a case